client/src/app/Main.res

```diff
@@ -1853,7 +1853,28 @@ let update_ = (msg: msg, m: model): modification => {
         Many(list{GetWorkerStatsAPICall(TL.id(tl)), getUnlockedDBs})
       | _ => getUnlockedDBs
       }
+
     | RefreshAvatars => ExpireAvatars
+
+    | RefreshPageIfUrlIsStable =>
```

Comment:
This is currently pretty naive - it doesn't check to see if it's on the latest client or not. It just refreshes every hour, if the page has a 'stable' URL and is not currently in focus.

Reply:
Are you sure it only does it if not in focus?

Reply:
I believe so - the `switch m.visibility {` in the branch seems to handle this fine for me locally.

Reply:
I think we don't know that is has been out of focus for an hour though?